images/macos/provision/configuration/configure-machine.sh

```diff
@@ -105,3 +105,5 @@ if [ ! -d "/usr/local/bin" ];then
 fi
 chmod +x $HOME/utils/invoke-tests.sh
 sudo ln -s $HOME/utils/invoke-tests.sh /usr/local/bin/invoke_tests
+
+plutil -replace SuggestionsAppLibraryEnabled -bool NO /Users/runner/Library/Developer/CoreSimulator/Devices/device_set.plist
```

Comment:
nitpick: replace /Users/runner to $HOME due to remove hardcoded path with username.